<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="generacja">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Generacja mapy</string>
    </property>
    <widget class="QSpinBox" name="spinRozmiar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinTunele">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinPokoje">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelPokoje">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pokoje</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTunele">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Długość tuneli</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRozmiar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rozmiar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
